docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">    # these two calls are equivalent 
    az.plot_trace(results, var_names=['Intercept', 'condition']) 
</pre><p>I think this belongs to a different problem? There's no <code>condition</code> variable  in <code>o + c + e + a + n</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>